practice-two/src/types/studentProfile.ts

```diff
@@ -0,0 +1,18 @@
+export interface StudentProfile {
+    name: string;
+    studentID: string;
+    email: string;
+    classes: string;
+    gender: string
+    phone: number;
+    password: string;
+    imageUrl: string;
+    age: number;
+}
+
+export interface IStudentInformation {
```

Comment:
Do you see all field IStudentInformation have contain in StudentProfile. please define and reuse